### PR TITLE
Add attack/happyPath and asset information

### LINKED ISSUE #15699: Complete SecurityTest information
PR #15372 replaced the `TEST_EVIDENCE` comments used by the `EvidenceSecurity` tool with the test-security framework.

More information is required by the new SecurityTest tags, such as "asset", "attack" and "happyPath" which is written to the CSV and JSON files. This information was not available before and needs to be added.

Part of: https://github.com/digital-asset/daml/issues/14254

### PR DESCRIPTION
**Pull request by cbley-da**
Fixes #15699

### CODE REVIEW

File: triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala [hunk 1]
```diff
@@ -501,7 +501,10 @@ trait AbstractTriggerServiceTest extends AbstractTriggerServiceTestHelper {
     } yield succeed
   }
 
-  it should "restart trigger on initialization failure due to failed connection" taggedAs availabilitySecurity inClaims withTriggerService(
+  it should "restart trigger on initialization failure due to failed connection" taggedAs availabilitySecurity
+    .setHappyCase(
```
Comment (matthiasS-da) [inline triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala:505]: I think this is more about reliability than security. Could you please change the tags, so this ends up in the right bucket?

File: triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala [hunk 2]
```diff
@@ -520,7 +523,10 @@ trait AbstractTriggerServiceTest extends AbstractTriggerServiceTestHelper {
     } yield succeed
   }
 
-  it should "restart trigger on run-time failure due to dropped connection" taggedAs availabilitySecurity inClaims withTriggerService(
+  it should "restart trigger on run-time failure due to dropped connection" taggedAs availabilitySecurity
```
Comment (matthiasS-da) [inline triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala:526]: Same here. Please double-check all usages of `availabilitySecurity`.

**Review by matthiasS-da — APPROVED** [review]
Pre-approving. Please address my comments before merging.

File: ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTestUserManagement.scala [hunk 3]
```diff
@@ -78,89 +79,101 @@ class HttpServiceIntegrationTestUserManagementNoAuth
...
-          None,
-          Some(NonEmptyList(bob)),
-          None,
-          submissionId = None,
-          deduplicationPeriod = None,
+    "should fail if overwritten actAs & readAs result in missing permission even if the user would have the rights" taggedAs authorizationSecurity
+      .setAttack(
+        Attack(
+          "Ledger client",
+          "tries to create an IOU but actAs & readAs result in missing permission",
```
[the remark this answers is not in the document]
Comment (cbley-da) [reply ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTestUserManagement.scala]: Done.

File: triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala (hunk 1 above)
Comment (matthiasS-da) [reply triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala:505]: Yes, that's fine!

Comment (cbley-da) [reply triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala:505]: As discussed, going to merge without changing to `ReliabilityTest`.

File: triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala (hunk 2 above)
Comment (cbley-da) [reply triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala:526]: Likewise.

**Pull request merged**